apps/studio/components/interfaces/Settings/Logs/LogsPreviewer.tsx

```diff
@@ -229,10 +230,18 @@ export const LogsPreviewer = ({
             <LogEventChart
               data={eventChartData}
               onBarClick={(isoTimestamp) => {
+                // from should be 1 hour before the bar
+                // to should be 1 hour after the bar
```

Comment:
not sure about this approach tbh

problem is, with current approach sometimes you click a bar and dont see the logs, because in the chart the error is at 11:12 and the timestamp you limited to was 11:00.

Reply:
1 hour diff might be a bit big if we're mitigating the second step differential, maybe just step back by 1 sec is enough

Reply:
Would require knowing what interval type it is at though, if it's minutes then would need to step back by a min and so on...